### PR TITLE
docs: Add Eric Sim Chong Li to `AboutUs`

### PR DESCRIPTION
**Pull request by e0958902**

### CODE REVIEW

File: docs/AboutUs.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
---------|:----:|:--------------:|:---------:
-![](https://avatars.githubusercontent.com/u/33289235) | Hoe Wen En | [Github](https://github.com/wenenhoe) | [Portfolio](team/wenenhoe.md)
-![](https://via.placeholder.com/100.png?text=Photo) | Don Joe | [Github](https://github.com/) | [Portfolio](docs/team/johndoe.md)
-![](https://via.placeholder.com/100.png?text=Photo) | Ron John | [Github](https://github.com/) | [Portfolio](docs/team/johndoe.md)
-![](https://via.placeholder.com/100.png?text=Photo) | John Roe | [Github](https://github.com/) | [Portfolio](docs/team/johndoe.md)
-![](https://via.placeholder.com/100.png?text=Photo) | Don Roe | [Github](https://github.com/) | [Portfolio](docs/team/johndoe.md)
+Display |       Name        | Github Profile | Portfolio 
+--------|:-----------------:|:--------------:|:---------:
+![](https://avatars.githubusercontent.com/u/33289235) |    Hoe Wen En     | [Github](https://github.com/wenenhoe) | [Portfolio](team/wenenhoe.md)
+![](https://via.placeholder.com/100.png?text=Photo) | Eric Sim Chong Li | [Github](https://github.com/e0958902) | [Portfolio](docs/team/johndoe.md)
```
Comment (annoy-o-mus) [inline docs/AboutUs.md:6]: Please update the profile picture to that of yours.

**Review by annoy-o-mus — CHANGES_REQUESTED** [review]

**Comment by annoy-o-mus** [comment]
@T0nyLin Need your help to check as well to make sure all is in order

File: docs/AboutUs.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
---------|:----:|:--------------:|:---------:
-![](https://avatars.githubusercontent.com/u/33289235) | Hoe Wen En | [Github](https://github.com/wenenhoe) | [Portfolio](team/wenenhoe.md)
-![](https://via.placeholder.com/100.png?text=Photo) | Don Joe | [Github](https://github.com/) | [Portfolio](docs/team/johndoe.md)
-![](https://via.placeholder.com/100.png?text=Photo) | Ron John | [Github](https://github.com/) | [Portfolio](docs/team/johndoe.md)
-![](https://via.placeholder.com/100.png?text=Photo) | John Roe | [Github](https://github.com/) | [Portfolio](docs/team/johndoe.md)
-![](https://via.placeholder.com/100.png?text=Photo) | Don Roe | [Github](https://github.com/) | [Portfolio](docs/team/johndoe.md)
+Display |       Name        | Github Profile | Portfolio 
+--------|:-----------------:|:--------------:|:---------:
+![](https://avatars.githubusercontent.com/u/33289235) |    Hoe Wen En     | [Github](https://github.com/wenenhoe) | [Portfolio](team/wenenhoe.md)
+![](https://avatars.githubusercontent.com/u/156929749) | Eric Sim Chong Li | [Github](https://github.com/e0958902) | [Portfolio](team/johndoe.md)
```
Comment (T0nyLin) [inline docs/AboutUs.md:6]: Please add your own markdown portfolio

Comment (e0958902) [reply docs/AboutUs.md]: Resolved in 98cdb0a.

**Review by T0nyLin — APPROVED** [review]
No more merge conflicts

**Review by annoy-o-mus — APPROVED** [review]
LGTM!

**Pull request merged**